src/test/resources/csv/thumbs.csv

```diff
@@ -0,0 +1,6 @@
+Project Name,Item ARK,Parent ARK,Object Type,File Name,Item Sequence,Type.typeOfResource,Type.genre,Rights.copyrightStatus,Rights.servicesContact,Language,Name.repository,Title,AltIdentifier.local,Description.note,Place of origin,Format.dimensions,Date.creation,Date.normalized,Summary,AltTitle.uniform,Bucketeer State,IIIF Access URL,thumbnail
+Hathaway Manuscripts,ark:/21198/zz0009gsq9,,Collection,,,,,pd,"UCLA Charles E. Young Research Library Department of Special Collections, A1713 Young Research Library, Box 951575, Los Angeles, CA 90095-1575. E-mail: spec-coll@library.ucla.edu. Phone: (310)825-4988",lat,"University of California, Los Angeles. Library. Performing Arts Special Collections",Hathaway Manuscripts,,,Germany|~|Switzerland|~|Low Countries,,11th - 15th century,1001/1500,"Music manuscript fragments used in bindings. Copied in Switzerland, Germany and the Low Countries. Contents from Missal, Gradual, Hymnal, Breviary and Antiphonal.",Hathaway collection of music manuscript leaves,succeeded,,"https://iiif.library.ucla.edu/iiif/2/ark%3A%2F21198%2Fzz002czs4c/full/600,/0/default.jpg"
+Hathaway Manuscripts,ark:/21198/zz0009gv8j,ark:/21198/zz0009gsq9,Work,,1,text,Manuscripts,pd,"UCLA Charles E. Young Research Library Department of Special Collections, A1713 Young Research Library, Box 951575, Los Angeles, CA 90095-1575. E-mail: spec-coll@library.ucla.edu. Phone: (310)825-4989",lat,"University of California, Los Angeles. Library. Performing Arts Special Collections",Hathaway 1,170/768 v.1,"Breviary: Readings and Responsories.|~|Written in the Alemannic area, presumably in Switzerland.|~|Parchment (rather coarse), 13 long lines; ruling not discernible; pricking in the outer margin.  Written in a late, crude Caroline minuscule with squarish Alemannic features; dark brown ink.  Plain initials in red or stroked with red; initials in the text slashed with yellow; rubrics in red majuscule; neumes of St. Gall type.|~|Used for binding; folio 1 used as a pastedown: the verso is stained and rubbed.|~|ff. 1, 2 and 3 from the same manuscript.",Switzerland ?,202 x 135 mm,11th century,1001/1100,,,failed,,"https://iiif.library.ucla.edu/iiif/2/ark%3A%2F21198%2Fzz002czs4c/full/600,/0/default.jpg"
```

Comment:
Why do the thumbnails in this file have full IIIF paths?

Reply:
Forgot to update when switched to base URLs